### PR TITLE
Workerize

### PR DESCRIPTION
**Pull request by kvigen**
This change adds the standard files for a worker repo
- Makefile
- drone.yml
- launch config
- Dockerfile
- readme

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM google/debian:wheezy
+
+RUN apt-get -y update && apt-get -y install curl
+RUN curl -L https://github.com/Clever/gearcmd/releases/download/v0.3.6/gearcmd-v0.3.6-linux-amd64.tar.gz | tar xz -C /usr/local/bin --strip-components 1
```
Comment (natebrennand) [inline Dockerfile]: any reason to not use `0.3.8` since it is available?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+`--speed`     | `1`          | Number of operations per second
+`--mongoURL`  | `localhost`  | Mongo URL to run the operations against
+`--path`      | `/dev/stdin` | Oplog file to replay
+
+
+## Development
+You can run locally with:
+```
+make
+go run main.go --port 6000
```
Comment (natebrennand) [inline README.md]: This seems like it was copied from something else by accident

**Comment by natebrennand** [comment]
lgtm minus the two nits and the broken build

File: Dockerfile (hunk 1 above)
Comment (kvigen) [inline Dockerfile]: nope, just copied and pasted `0.3.6` over

File: README.md (hunk 2 above)
Comment (kvigen) [inline README.md]: good catch

**Pull request merged**